specifyweb/frontend/js_src/lib/components/Interactions/InteractionDialog.tsx

```diff
@@ -86,6 +86,12 @@ export function InteractionDialog({
           readonly entries: RA<PreparationData>;
         }
       >
+    | State<
+        'UnavailableState',
```

Comment:
just a fyi: I used to have the name of each state end with `*State` to clearly differentiate them, but am not doing that anymore as it is redundant pretty often (it's clear that state name is being referred to when you see state.type in the code; adding `*State` is only adding verbosity)

not changes needed here; just a FYI for the future